assets/src/blocks/Media/MediaEditor.js

```diff
@@ -17,10 +17,24 @@ const MediaInspectorOptions = ({ attributes, setAttributes }) => {
   // https://github.com/facebook/react/issues/14904
   // https://github.com/facebook/react/issues/955#issuecomment-469352730
   const [ media_url, setMediaURL ] = useState(attributes.media_url);
-  const debouncedMediaURLUpdate = useCallback(debounce(value => setAttributes({ media_url: value }), 300), []);
+  const debouncedMediaURLUpdate = useCallback(debounce(value => {
+    setAttributes({ embed_html: null });
```

Comment:
Why the need to set `null` here?

Reply:
We probably don't, I added it so that the previous embed html is already cleared before the api request to get the embed is completed, but I guess we can just leave the previous html in place until the new arrives.